README.md

```diff
@@ -1,109 +1,66 @@
 ![next branch build status](https://travis-ci.org/freenet/fred.svg?branch=next)
 ![Coverity status](https://scan.coverity.com/projects/2316/badge.svg?flat=1)
 
-## Quickstart
+# Freenet
 
-To install Freenet, use the installer from [Freenet Project](https://freenetproject.org)
+Freenet is a platform for censorship-resistant communication and publishing. It is peer-to-peer
+software which provides a distributed, encrypted, decentralized datastore. Websites and applications
+providing things like forums and chat are built on top of it.
 
-If the installer did not open a browser with Freenet, start Freenet and access the
-Freenet web interface at http://127.0.0.1:8888/
+Fred stands for Freenet REference Daemon.
+
+## Building
+
+### Bundled
+
+We've included the [Gradle Wrapper](https://docs.gradle.org/3.2/userguide/gradle_wrapper.html) as
+recommended by the Gradle project. If you trust the version we've committed you can build
+immediately:
+
+#### POSIX-based:
+
+    $ ./gradlew jar
+
+#### Windows:
+
+    > gradlew.bat jar
```

Comment:
I guess we can tell them to use PowerShell for the purposes of these directions.